src/prism.c

```diff
@@ -21905,6 +21905,7 @@ wrap_statements(pm_parser_t *parser, pm_statements_node_t *statements) {
             ));
 
             pm_arguments_node_arguments_append(arguments, (pm_node_t *) keywords);
+            pm_node_flag_set((pm_node_t *) arguments, PM_ARGUMENTS_NODE_FLAGS_CONTAINS_KEYWORDS);
```

Comment:
Wondering if it's better to set this flag in `pm_arguments_node_arguments_append`

Reply:
That's okay, I think this is fine here.